editoast/src/views/infra/objects.rs

```diff
@@ -114,6 +119,47 @@ async fn get_objects(
     Ok(Json(result))
 }
 
+#[derive(serde::Serialize, utoipa::ToSchema)]
+struct ListObjectsResponse {
+    ids: Vec<String>,
+}
+
+#[utoipa::path(
+    get, path = "",
+    tag = "infra",
+    params(InfraIdParam, ObjectTypeParam),
+    responses(
+        (status = 200, description = "The list of objects", body = ListObjectsResponse),
```

Comment:
```suggestion
        (status = 200, description = "The list of objects", body = inline(ListObjectsResponse)),
```

or 

```suggestion
        (status = 200, description = "The list of objects", body = Vec<String>),
```

directly

Reply:
Generally, it's a "good idea" to return json objects as a top level response... (To allow adding stuff later without making it a breaking change) But i could go either way, wdyt

Reply:
I don't mind an object at all, but in this case since the schema is only used at this one place better inline it